README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Kieran Coffey
+I am a MAPSS student studying European History - specifically German cultural and political history of the nineteenth and twentieth centuries. At the moment I am studying the relationship between conceptions of regional, national, and religious identity as they relate to the German concept of *Heimat*, or homeland. My other academic interests include:
+
+    - nationalism and national identity
+    - sub-national or regional identity
+    - mass political movements, particularly socialist/communist and far-right movements
+    - borderland regions, particular between Germany and France 
+    and the European Union 
+
+## Outside of Academia
+I also like to hike, watch soccer, play video games (especially grand stratedgy games! <https://www.paradoxplaza.com>), trying new recipes, and visiting friends in other cities and countries.  
```

Comment:
This is super minor and you might not even want to do it, but if you are interested there are ways to format your body paragraphs to add space / indent.